### PR TITLE
Update README to aid local development for WS sites

### LINKED ISSUE #1816: Update README with Front Page routes
**Is your feature request related to a problem? Please describe.**
Similar to how article examples have been given [here](https://github.com/bbc/simorgh#local-development), we should document how to access a front page route.

**Describe the solution you'd like**
I'd like to be able to understand how I can locally access a canonical and amp front page route.

**Testing notes**
[Tester to complete]

Dev insight: No code change. Sanity check on the `README` once updated.

### PR DESCRIPTION
**Pull request by 12**
Resolves #1816 

**Overall change:** Updates `README.md` to document how to access World Service front pages locally.

**Code changes:**

- Updates `README.md`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by DenisHdz — DISMISSED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ To run this application locally, with hot-reloading, run
 npm run dev
 ```
 
-The application will start on [http://localhost:7080](http://localhost:7080). Article pages are served at routes of the format  `/news/articles/:id` where id is the asset ID generated by the Content Management System. 
+The application will start on [http://localhost:7080](http://localhost:7080). 
+
+Article pages are served at routes of the format  `/news/articles/:id` where id is the asset ID generated by the Content Management System.
```
Comment (andrew-nowak) [inline README.md:46]: Thanks for updating the readme! It might be worth adding documentation that persian articles can also be accessed locally

**Review by dr3 — CHANGES_REQUESTED** [review]
https://github.com/bbc/simorgh/pull/1844#discussion_r290208470

**Review by dr3 — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
Prost!

**Comment by david-boydell** [comment]
Looks good.

**Pull request merged**